api/registry/api.py

```diff
@@ -243,3 +262,50 @@ def get_score(request, address: str, community_id: int) -> ScoreResponse:
             exc_info=True,
         )
         raise InvalidScoreRequestException()
+
+
+class ScoreFilters(Schema):
+    community_id: int
+    limit: int = 50
+    offset: int = 0
+    addresses: List[str] = Field(None, alias="addresses")
+
+
+@router.get("/scores", auth=ApiKey(), response=List[ScoreResponse])
+def get_scores(request, filters: ScoreFilters = Query({})) -> List[ScoreResponse]:
```

Comment:
In my opinion this should be a read function that returns a list of items, and allows filtering, and we should keep it somewhat generic.
So:
1.  the filter attributes addresses and community_id should be optional
2. if no filter is specified, it is ok to return the entire list of results
3. the current implementation does not restrict. I don't think this is an explicit requirement, but given that we use the api key, we should use this restriction.